include/boost/sml.hpp

```diff
@@ -516,7 +516,15 @@ class queue_event {
   queue_event(queue_event &&other) : id(other.id), dtor(other.dtor), move(other.move) {
     move(data, static_cast<queue_event &&>(other));
   }
-  queue_event &operator=(queue_event &&) = default;
+  queue_event &operator=(queue_event &&other) {
```

Comment:
Thanks, @palarson, would you be able to provide test to cover the issue and keep the code working in the future 👍